compiler/src/main/scala/org/scalaexercises/exercises/compiler/Compiler.scala

```diff
@@ -235,8 +235,9 @@ case class Compiler() {
         )
     }
 
-    def oneline(msg: String) = {
-      val msg0 = msg.lines.mkString(s"${Console.BLUE}\\n${Console.RESET}")
+    // TODO: Not being used anywhere. Can this be removed?
+    /*def oneline(msg: String) = {
```

Comment:
If it's not needed, let's remove this commented function along with the TODO comment